### PR TITLE
Remove the condition for location accuracy to reach threshold before adding tree

### PR DESCRIPTION
**Pull request by arunbakt**
Now that we use threshold value for variance of contiguous location data stream to determine when it is fine to proceed with adding a tree (enabling the camera to take a picture), we don't need to use accuracy also. This code was left in prior updates that introduced variance of location data. This PR is clean up to remove those references in map fragment.

### CODE REVIEW

File: app/src/main/java/org/greenstand/android/TreeTracker/fragments/MapsFragment.kt [hunk 1]
```diff
@@ -50,7 +44,10 @@ import org.koin.android.ext.android.inject
 import org.koin.android.viewmodel.ext.android.viewModel
 import timber.log.Timber
 
-class MapsFragment : androidx.fragment.app.Fragment(), OnClickListener, OnMapReadyCallback,
+class MapsFragment :
+    androidx.fragment.app.Fragment(),
+    OnClickListener,
+    OnMapReadyCallback,
```
Comment (arunbakt) [inline app/src/main/java/org/greenstand/android/TreeTracker/fragments/MapsFragment.kt:50]: ignore, ktlint formatting change

File: app/src/main/java/org/greenstand/android/TreeTracker/fragments/MapsFragment.kt [hunk 2]
```diff
@@ -74,34 +70,36 @@ class MapsFragment : androidx.fragment.app.Fragment(), OnClickListener, OnMapRea
...
         convergenceProgressView.visibility = View.GONE
 
-        vm.checkInStatusLiveData.observe(this, Observer<Boolean> { planterIsCheckedIn ->
-            if (planterIsCheckedIn) {
-                GlobalScope.launch(Dispatchers.Main) {
+        vm.checkInStatusLiveData.observe(
+            this,
+            Observer<Boolean> { planterIsCheckedIn ->
+                if (planterIsCheckedIn) {
+                    GlobalScope.launch(Dispatchers.Main) {
```
Comment (arunbakt) [inline app/src/main/java/org/greenstand/android/TreeTracker/fragments/MapsFragment.kt:80]: ignore ktlint formatting

File: app/src/main/java/org/greenstand/android/TreeTracker/fragments/MapsFragment.kt [hunk 3]
```diff
@@ -74,34 +70,36 @@ class MapsFragment : androidx.fragment.app.Fragment(), OnClickListener, OnMapRea
...
+                        val photoPath = user.profilePhotoPath
+                        val profileImageView = mapUserImage
 
-                    if (photoPath != null) {
-                        val rotatedBitmap =
-                            ImageUtils.decodeBitmap(photoPath, resources.displayMetrics.density)
-                        if (rotatedBitmap != null) {
-                            profileImageView.setImageBitmap(rotatedBitmap)
-                            profileImageView.visibility = View.VISIBLE
+                        if (photoPath != null) {
```
Comment (arunbakt) [inline app/src/main/java/org/greenstand/android/TreeTracker/fragments/MapsFragment.kt:87]: Ignore, ktlint formatting

File: app/src/main/java/org/greenstand/android/TreeTracker/models/FeatureFlags.kt [hunk 4]
```diff
@@ -3,13 +3,16 @@ package org.greenstand.android.TreeTracker.models
 import org.greenstand.android.TreeTracker.BuildConfig
 
 object FeatureFlags {
-    val DEBUG_ENABLED: Boolean = BuildConfig.BUILD_TYPE == "dev" || BuildConfig.BUILD_TYPE == "debug" || BuildConfig.BUILD_TYPE == "acceleration"
+    val DEBUG_ENABLED: Boolean = (
```
Comment (arunbakt) [inline app/src/main/java/org/greenstand/android/TreeTracker/models/FeatureFlags.kt:6]: ktlint format change, please ignore

**Review by Elforama — APPROVED** [review]

**Pull request merged**